### PR TITLE
Fix cleanup instructions for TLS version config task

### PR DESCRIPTION
**Pull request by SRodi**
Please provide a description for what this PR is for.

The cleanup instructions for **Istio Workload Minimum TLS Version Configuration** are not complete as they do not currently include the delete of sample applications and the uninstall of Istio.

This PR relates to #12727.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by kfaseela** [comment]
Please run `make gen` locally to get the auto-gen files and commit it along with your changes

File: content/en/docs/tasks/security/tls-configuration/workload-min-tls-version/index.md [hunk 1]
```diff
@@ -86,6 +86,19 @@ Cipher is (NONE)
 
 ## Cleanup
 
+Delete sample applications `sleep` and `httpbin` from `foo` namespace:
```
Comment (ericvn) [inline content/en/docs/tasks/security/tls-configuration/workload-min-tls-version/index.md:89]:
```suggestion
Delete sample applications `sleep` and `httpbin` from the `foo` namespace:
```

Note that these instructions are not necessarily required as deleting the namespaces below will also delete the applications.

File: content/en/docs/tasks/security/tls-configuration/workload-min-tls-version/index.md [hunk 2]
```diff
@@ -86,6 +86,19 @@ Cipher is (NONE)
...
+
+{{< text bash >}}
+$ kubectl delete -f samples/httpbin/httpbin.yaml -n foo
+$ kubectl delete -f samples/sleep/sleep.yaml -n foo
+{{< /text >}}
+
+Uninstall Istio from cluster:
+
+{{< text bash >}}
+$ istioctl uninstall --revision=default
```
Comment (ericvn) [inline content/en/docs/tasks/security/tls-configuration/workload-min-tls-version/index.md:99]: It does look like the test runs this command, but you might change to the more common `istioctl uninstall --purge -y` in the instructions and the test.

File: content/en/docs/tasks/security/tls-configuration/workload-min-tls-version/index.md [hunk 3]
```diff
@@ -86,6 +86,19 @@ Cipher is (NONE)
...
 ## Cleanup
 
+Delete sample applications `sleep` and `httpbin` from `foo` namespace:
+
+{{< text bash >}}
+$ kubectl delete -f samples/httpbin/httpbin.yaml -n foo
+$ kubectl delete -f samples/sleep/sleep.yaml -n foo
+{{< /text >}}
+
+Uninstall Istio from cluster:
```
Comment (ericvn) [inline content/en/docs/tasks/security/tls-configuration/workload-min-tls-version/index.md:96]:
```suggestion
Uninstall Istio from the cluster:
```

**Review by johnma14 — COMMENTED** [review]
Changes look good. Thanks Simone.

File: content/en/docs/tasks/security/tls-configuration/workload-min-tls-version/index.md (hunk 1 above)
Comment (SRodi) [reply content/en/docs/tasks/security/tls-configuration/workload-min-tls-version/index.md]: addressed as part of [363e5ae](https://github.com/istio/istio.io/pull/12735/commits/363e5aed9e6a27a8875701ad61e2e51bf6c03142)

File: content/en/docs/tasks/security/tls-configuration/workload-min-tls-version/index.md (hunk 3 above)
Comment (SRodi) [reply content/en/docs/tasks/security/tls-configuration/workload-min-tls-version/index.md]: addressed as part of [363e5ae](https://github.com/istio/istio.io/pull/12735/commits/363e5aed9e6a27a8875701ad61e2e51bf6c03142)

**Review by johnma14 — COMMENTED** [review]
Looks great. Thanks Simone.

**Review by ericvn — APPROVED** [review]

**Pull request merged**